README.md

```diff
@@ -105,20 +86,11 @@ Note you need a computer with a CUDA-capable GPU to train the model.
 
 ## Generating the Answer Alignments for NewsQA-es
 
-Run these commands to generate the answer alignments for the NewsQA-es dataset. You should have a trained Mask-Align 
+Run the following script to generate the answer alignments for the NewsQA-es dataset. You should have a trained Mask-Align 
 model and have the `newsqa.csv` file.
 
 ```bash
-./scripts/generate-alignments/remove_bad_rows.py
-./scripts/generate-alignments/generate_files.py
-spm_encode --model=en.model --output_format=piece < test.en > test.32k.en  
-spm_encode --model=es.model --output_format=piece < test.es > test.32k.es
-./scripts/generate-alignments/process_answer_indexes.py
-mkdir corpus-es
-mv test.32k.en test.32k.es answers.en corpus-es/
-./thualign/bin/generate.sh -s spanish -o output.txt
-spm_decode --model=es.model --input_format=piece < output.txt > output-plain.txt
-./scripts/generate-alignments/output_brackets_to_indexes.py
+./scripts/generate-alignments/generate_alignments.sh
```

Comment:
Can you add the next part ("Generate the final merged CSV file") into this script as well?